doc/classes/Skeleton3D.xml

```diff
@@ -190,6 +212,10 @@
 			<argument index="1" name="custom_pose" type="Transform">
 			</argument>
 			<description>
+				Sets the custom pose transform, [code]custom_pose[/code], for the bone at [code]bone_idx[/code]. This pose is an addition to the bone rest pose.
+
```

Comment:
There should not be empty lines, nor cosmetic line breaks (line on line 218) in the XML. Each line == a paragraph.

Reply:
This applies to several of the descriptions below.